src/freenet/support/FileLoggerHook.java

```diff
@@ -636,7 +636,7 @@ public void findOldLogFiles(GregorianCalendar gc) {
 					name = name.substring(0, name.length()-".log.gz".length());
 				}
 				name = name.substring(prefix.length());
-				if((name.length() == 0) || (name.charAt(0) != '-')) {
+				if((name.isEmpty()) || (name.charAt(0) != '-')) {
```

Comment:
Parens, below as well.

Reply:
Fixed in ed98d8c127320b71b507f1524cce60e0d5efa356